src/utilities/sorting/sorting.test.ts

```diff
@@ -0,0 +1,12 @@
+import {initSortedIndexMapFunc} from "./sorting";
+
+describe("table functions", () => {
+    test("initialize sorted index map", () => {
+        const rowNumber: number = 4;
+        const list: number[] = [];
+        for (let i = 0; i < rowNumber; i++) {
+            list.push(i);
+        }
+        expect(initSortedIndexMapFunc(rowNumber)).toEqual(list);
```

Comment:
I think we can simplify it to `... toEqual([0, 1, 2, 3])`.

Reply:
I suggest we adjust the description to `describe("getInitIndexMap", () => { test("returns the correct indexes", ...` to show that we are testing only the function.